### ISSUE TITLE
Connection problem

### ISSUE THREAD

**Issue by georgpfolz** [jnimmo/pyIntesisHome#21]
When trying to connect (`await controller.connect()`), I get the following error message:

```
Connection to 212.36.84.207:5220 failed with exception module 'socket' has no attribute 'TCP_USER_TIMEOUT'
```

I still can query informations with methods like get_devices, get_power_state and so on, but the control methods do not seem to have any effect.

**Comment by jnimmo** [comment]
What host OS are you running? We may need to add a check to see if TCP timeout is available as an option, and to only set it if is

**Comment by georgpfolz** [comment]
I get this error message on MacOS (10.15.6) and on FreeBSD (12.1-RELEASE-p6).

**Comment by georgpfolz** [comment]
This might be useful:

- https://stackoverflow.com/questions/22515349/what-is-the-setsockopt-option-name-of-tcp-user-timeout-on-osx
- https://www.manpagez.com/man/4/tcp/

**Comment by georgpfolz** [comment]
I found some information that in MacOs and BSD TCP_USER_TIMEOUT can be replaced with TCP_CONNECTIONTIMEOUT, but this didn't prove to be true (`dir(socket)` reveals that it does not exist).

So I tried to patch the code, just commenting out the line in question. My script now runs without any error messages, but the control methods still seem to have no effect. 

Is it necessary to register an application somehow with IntesisHome (like IFTTT does)?
